projects/largo/umbrella-setup.md

```diff
@@ -187,27 +211,7 @@ You can name the snapshot anything you want, and I would recommend describing it
 
 We'll be doing some work soon of searching through the database we just loaded and replacing certain values. To make this easier, download version 2.1.0 of this [database search and replace script written in PHP](https://interconnectit.com/products/search-and-replace-for-wordpress-databases/). Unzip it and move the file `searchreplacedb2.php` to your `largo-umbrella` folder.
 
-## 12. Base configuration of WordPress.
-
-We need to tell WordPress where to look for the database and how to access it on the virtual machine.
-
-1. Go to [http://vagrant.dev](http://vagrant.dev), which should automatically redirect you to the config page of [http://vagrant.dev/wp-admin/setup-config.php](http://vagrant.dev/wp-admin/setup-config.php).
-
-2. Choose your language and then go to the next screen.
-
-3. Enter the following details for the database settings:
-
-    * Database Name: `largoproject`
-    * User Name: `root`
-    * Password: `root`
-    * Database Host: `localhost`
-    * Table Prefix: `wp_`
-
-4. Go to the next screen, where you have only one button to click, `Run the install`. Click that, and it will create the file `wp-config.php` in your `largo-umbrella` folder.
-
-5. You may then see the message "Already Installed" because you already loaded the WordPress database in a previous step.
-
-## 13. Changing largoproject.wpengine.com to vagrant.dev.
+## 12. Changing largoproject.wpengine.com to vagrant.dev.
```

Comment:
Should this section also include the `wp` options seen here: https://github.com/INN/docs/blob/master/projects/largo/database-reload.md#4-replace-primary-wp_-tables-with-local-environment-urls